packages/mobile/src/notifications/types.ts

```diff
@@ -16,7 +16,7 @@ export interface TransferNotificationData {
 }
 
 export enum NotificationReceiveState {
-  APP_ALREADY_OPEN,
-  APP_FOREGROUNDED,
-  APP_OPENED_FRESH,
+  APP_ALREADY_OPEN = 'APP_ALREADY_OPEN',
+  APP_FOREGROUNDED = 'APP_FOREGROUNDED',
+  APP_OPENED_FRESH = 'APP_OPENED_FRESH',
```

Comment:
Foregrounded sounds weird too since that case is actually for when the app is opened from the background. I guess it's grammatically correct but still confusing imo. I'll rename both and adapt to enums styleguide.